roles/redis/defaults/main.yml

```diff
@@ -0,0 +1 @@
+redis_image: docker.io/redis:6
```

Comment:
I think we should change this to a centos/stream9 container even if we have to build it ourselves. That could be logged as an issue and done as a follow up change.

Reply:
https://github.com/theforeman/foreman-quadlet/issues/94